src/libraries/Native/Unix/System.Native/browser/pal_random_browser.h

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "pal_compiler.h"
+#include "pal_types.h"
+
+PALEXPORT int32_t dotnet_browser_entropy(uint8_t* buffer, int32_t bufferLength);
```

Comment:
This should not be `PALEXPORT`. The method is not exported from the PAL.

Reply:
very true.  addressed in recent commits